index.bs

```diff
@@ -7052,6 +7052,15 @@ the [=[RP]=] could mitigate the privacy leak using the same approach of returnin
 as discussed in [[#sctn-username-enumeration]].
 
 
+# Accessibility Considerations # {#sctn-accessiblility-considerations}
+
+[=User verification=]-capable [=authenticators=], whether [=roaming authenticators|roaming=] or [=platform authenticators|platform=], should offer users more than one user verification method.  For example, both fingerprint sensing and PIN entry. This allows for fallback to other user verification means if the selected one is not working for some reason. Note that in the case of [=roaming authenticators=], the authenticator and platform might work together to provide a user verification method such as PIN entry [[FIDO-CTAP]].
+
+[=Client platforms=], at [=registration=] time, should provide affordances for users to complete [=authorization gestures=] correctly. This could involve naming the authenticator, choosing a picture to associate with the device, or entering freeform text instructions (e.g., as a reminder-to-self).
```

Comment:
>[=Client platforms=], at [=registration=] time, should provide affordances for users to complete [=authorization gestures=] correctly.

Does this mean authorization gestures in future ceremonies? I first read this as applying only to registration ceremonies, not authentication ceremonies. Would something like this be in line with your intent?

```suggestion
[=Client platforms=], at [=registration=] time, should provide affordances for users to complete future [=authorization gestures=] correctly. This could involve naming the authenticator, choosing a picture to associate with the device, or entering freeform text instructions (e.g., as a reminder-to-self).
```